composer.json

```diff
@@ -25,7 +25,8 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.2",
```

Comment:
This all-in-one commit prevents a cherry-pick and destroys the history. These changes require [separated commits](https://github.com/laminas/.github/blob/master/CONTRIBUTING.md#working-on-a-patch):

* adding new feature
* changing unit tests
* changing the Composer config

Reply:
@froschdesign I've updated to use separate commits

Reply:
Much better because the changes are now easy to understand. Every pull request should look like this. 👍